docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
 # Insights
 
-> This [Module](/getting-started/glossary/#modules) allows the creation of custom
-> [Dashboards](/getting-started/glossary/#dashboards), each with any number of highly configurable
-> [Panels](/getting-started/glossary/#panels) for displaying data.
+> Insights is a module in Directus that allows rapid, no-code buildout of analytics dashboards. Regardless of your
```

Comment:
> module in Directus

I usually capitalize our extensions (eg: `Module`), but we should decide if that's something we want to do moving forward. Also, maybe we say "Module _of_ Directus"?

Reply:
I'd like to keep all our "custom names" capitalized, so it'll be a Module / Interface / Display etc etc

Reply:
Okay swag. I saw one instance in the docs where that wasn't the case and rolled with lowercase.
Def down to capitalize and had/have the impulse in my fingers to do the same.
(If I'm becoming the main tech docs geezer, I'm happy to shred them apart and touch up- once I know all the standards)